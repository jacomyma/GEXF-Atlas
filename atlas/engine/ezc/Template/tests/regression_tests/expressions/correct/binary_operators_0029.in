{var $foo = 42, $bar = 101, $bar2 = 201, $arr = array( 'foo' => 401 )}
0897:[{debug_dump( !( $foo !== 5 ) + 5 )}]
0898:[{debug_dump( !( $foo !== 5 ) + -$foo )}]
0899:[{debug_dump( !( $foo !== 5 ) - 5 )}]
0900:[{debug_dump( !( $foo !== 5 ) - -$foo )}]
0901:[{debug_dump( !( $foo !== 5 ) * 5 )}]
0902:[{debug_dump( !( $foo !== 5 ) * -$foo )}]
0903:[{str_number( !( $foo !== 5 ) / 5 , 2, ',', '' )}]
0904:[{str_number( !( $foo !== 5 ) / -$foo , 2, ',', '' )}]
0905:[{debug_dump( !( $foo !== 5 ) % 5 )}]
0906:[{debug_dump( !( $foo !== 5 ) % -$foo )}]
0907:[{debug_dump( !( $foo !== 5 ) == 5 )}]
0908:[{debug_dump( !( $foo !== 5 ) == -$foo )}]
0909:[{debug_dump( !( $foo !== 5 ) === 5 )}]
0910:[{debug_dump( !( $foo !== 5 ) === -$foo )}]
0911:[{debug_dump( !( $foo !== 5 ) != 5 )}]
0912:[{debug_dump( !( $foo !== 5 ) != -$foo )}]
0913:[{debug_dump( !( $foo !== 5 ) !== 5 )}]
0914:[{debug_dump( !( $foo !== 5 ) !== -$foo )}]
0915:[{debug_dump( !( $foo !== 5 ) < 5 )}]
0916:[{debug_dump( !( $foo !== 5 ) < -$foo )}]
0917:[{debug_dump( !( $foo !== 5 ) <= 5 )}]
0918:[{debug_dump( !( $foo !== 5 ) <= -$foo )}]
0919:[{debug_dump( !( $foo !== 5 ) > 5 )}]
0920:[{debug_dump( !( $foo !== 5 ) > -$foo )}]
0921:[{debug_dump( !( $foo !== 5 ) >= 5 )}]
0922:[{debug_dump( !( $foo !== 5 ) >= -$foo )}]
0923:[{debug_dump( !( $foo !== 5 ) && 5 )}]
0924:[{debug_dump( !( $foo !== 5 ) && -$foo )}]
0925:[{debug_dump( !( $foo !== 5 ) || 5 )}]
0926:[{debug_dump( !( $foo !== 5 ) || -$foo )}]
0927:[{debug_dump( !( $foo !== 5 ) . 5 )}]
0928:[{debug_dump( !( $foo !== 5 ) . -$foo )}]
