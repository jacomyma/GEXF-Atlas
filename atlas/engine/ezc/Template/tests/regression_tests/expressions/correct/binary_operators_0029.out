0897:[5]
0898:[-42]
0899:[-5]
0900:[42]
0901:[0]
0902:[0]
0903:[0,00]
0904:[0,00]
0905:[0]
0906:[0]
0907:[false]
0908:[false]
0909:[false]
0910:[false]
0911:[true]
0912:[true]
0913:[true]
0914:[true]
0915:[true]
0916:[true]
0917:[true]
0918:[true]
0919:[false]
0920:[false]
0921:[false]
0922:[false]
0923:[false]
0924:[false]
0925:[true]
0926:[true]
0927:['5']
0928:['-42']
